app/src/main/java/ca/llamabagel/transpo/ui/home/MainActivity.kt

```diff
@@ -39,6 +34,10 @@ class MainActivity : AppCompatActivity() {
         val navController = findNavController(R.id.navHostFragment)
         findViewById<BottomNavigationView>(R.id.bottomNavigation)
             .setupWithNavController(navController)
+
+        findViewById<EditText>(R.id.editText2).setOnClickListener {
```

Comment:
Maybe a more descriptive id for the EditText? Unless this is just temporary.

Reply:
you named it lol

Reply:
Oops